src/app/content/types.ts

```diff
@@ -115,3 +116,32 @@ export interface ArchivePage {
 }
 
 export type ArchiveContent = ArchivePage | ArchiveBook;
+
+export type HighlightData = Highlight;
```

Comment:
lets either move these to a file like `highlight-data-types.ts` that is shared between the two or just move them back into the highlighting types file.

they don't make a lot of sense here outside of the context of any highlighting stuff, and they don't apply to other content logic